### PR TITLE
Issue 58 solidity test code coverage

### LINKED ISSUE #58: compute code coverage for truffle tests
follow up by writing more tests that put coverage to 100%

### PR DESCRIPTION
**Pull request by snd**
resolves #58

### CODE REVIEW

**Comment by snd** [comment]
done. can i get a review?

**Comment by snd** [comment]
once this gets merged the CI for master should push to coveralls and the badge in the readme should work

File: .travis.yml [hunk 1]
```diff
@@ -22,10 +22,13 @@ matrix:
         - cargo test --all
     - language: node_js
       node_js: node
-      before_script: cd truffle
+      before_script:
+        - cd truffle
+        - npm install
```
Comment (tomusdrw) [inline .travis.yml]: Can we use `yarn` and commit `yarn.lock` as well for faster and deterministic builds?

**Comment by snd** [comment]
done : )

**Comment by snd** [comment]
can i have another look at this?

**Pull request merged**